config/database.yml

```
@@ -1,32 +1,55 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
+# # SQLite. Versions 3.8.0 and up are supported.
+# #   gem install sqlite3
+# #
+# #   Ensure the SQLite 3 gem is defined in your Gemfile
+# #   gem "sqlite3"
+# #
+# default: &default
+#   adapter: sqlite3
+#   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+#   timeout: 5000
+
+# development:
+#   <<: *default
+#   database: storage/development.sqlite3
+
+# # Warning: The database defined as "test" will be erased and
+# # re-generated from your development database when you run "rake".
+# # Do not set this db to the same as development or production.
+# test:
+#   <<: *default
+#   database: storage/test.sqlite3
+
+
+# # SQLite3 write its data on the local filesystem, as such it requires
+# # persistent disks. If you are deploying to a managed service, you should
+# # make sure it provides disk persistence, as many don't.
+# #
+# # Similarly, if you deploy your application as a Docker container, you must
+# # ensure the database is located in a persisted volume.
+# production:
+#   <<: *default
+#   # database: path/to/persistent/storage/production.sqlite3
+
+
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
+  username: nahok
+  password: 920518
+  host: localhost
```

Dejemos esto así, de lo contrario obligaras a cualquier persona con el proyecto a tener los mismos valores que tu. Además, el password lo estas quemando en este archivo, eso incrementa la vulnerabilidad, mientras que si usas el password en una variable de entorno nadie puede saber cual es tu contraseña

```suggestion
  host: <%= ENV.fetch("DB_HOST", "localhost") %>
  username: <%= ENV.fetch("DB_USERNAME", "postgres") %>
  password: <%= ENV.fetch("DB_PASSWORD", "") %>
```